upstream-community-operators/splunk/0.0.6/splunk-operator.v0.0.6.clusterserviceversion.yaml

```diff
@@ -0,0 +1,242 @@
+    generate. It gives you insights to drive operational performance and
+    business results.
+
+
+    Please refer to [Splunk
+    products](https://www.splunk.com/en_us/software.html) for more knowledge
+    about the features and capabilities of Splunk, and how you can bring it into
+    your organization.
+
+    Please see the [getting started
+    guide](https://splunk.github.io/splunk-operator/) to get up and running with
+    the Splunk Operator for Kubernetes.
+
+
+    * [Known Issues for the Splunk
+    Operator](https://splunk.github.io/splunk-operator/#known-issues-for-the-splunk-operator)
+
+    * [Prerequisites for the Splunk
+    Operator](https://splunk.github.io/splunk-operator/#prerequisites-for-the-splunk-operator)
+
+    * [Installing the Splunk
+    Operator](https://splunk.github.io/splunk-operator/#installing-the-splunk-operator)
+
+    * [Creating SplunkEnterprise
+    Deployments](https://splunk.github.io/splunk-operator/#creating-splunkenterprise-deployments)
+
+
+    COMMUNITY SUPPORTED: Splunk Operator for Kubernetes is an open source
+    product developed by Splunkers with contributions from the community of
+    partners and customers. This unique product will be enhanced, maintained and
+    supported by the community, led by Splunkers with deep subject matter
+    expertise. The primary reason why Splunk is taking this approach is to push
+    product development closer to those that use and depend upon it. This direct
+    connection will help us all be more successful and move at a rapid pace.
+
+
+    For help, please first read our [Frequently Asked
+    Questions](https://splunk.github.io/splunk-operator/FAQ.html)
+
+
+    Community Support & Discussions on
+    [Slack](https://splunk-usergroups.slack.com/) channel #splunk-operator
+
+
+    File Issues or Enhancements in
+    [GitHub](https://github.com/splunk/splunk-operator/issues)
+    splunk/splunk-operator
+
+    Please see [Prerequisites for the Splunk
+    Operator](https://splunk.github.io/splunk-operator/#prerequisites-for-the-splunk-operator)
+
+    The Splunk Operator for Kubernetes uses a `SplunkEnterprise` resource to
+    create a single deployment of Splunk Enterprise. For example, run the
+    following command to create a single instance deployment named “s1”:
+
+
+    ```
+
+    cat <<EOF | kubectl apply -f -
+
+    apiVersion: enterprise.splunk.com/v1alpha1
+
+    kind: SplunkEnterprise
+
+    metadata:
+      name: s1
+      finalizers:
+      - enterprise.splunk.com/delete-pvc
+    EOF
+
+    ```
+
+
+    The `SplunkEnterprise` resource supports many additional configuration
+    parameters. Please see [SplunkEnterprise
+    Parameters](https://splunk.github.io/splunk-operator/SplunkEnterprise.html)
+    for more information. For more deployment examples, please see [Configuring
+    Splunk Enterprise
+    Deployments](https://splunk.github.io/splunk-operator/Examples.html).
+
+  maturity: alpha
+  version: 0.0.6
+  replaces: ''
```

Comment:
Please remove this line as it is the new operator.

Reply:
Removed